### ISSUE TITLE
CTC loss

### ISSUE THREAD

**Issue by leigaoxiang** [clovaai/deep-text-recognition-benchmark#129]
when i replace torch.nn.CTCloss to warp-ctc loss ,it becomes so low when training

**Comment by ku21fan** [comment]
Hello, 

Check torch.nn.CTCloss, which needs [log_softmax  before the loss calculation](https://github.com/clovaai/deep-text-recognition-benchmark/blob/master/train.py#L145). 
So if you remove log_softmax, it would be fine. 
or just check our old commit 2b38eb25749519c9a3646e19628cd7b6a398043d which used [torch_baidu_ctc](https://pypi.org/project/torch-baidu-ctc/) (= Pytorch bindings for Baidu's Warp-CTC)

Best.
